User/Config/Dolphin.ini

```diff
@@ -1,192 +1,192 @@
-ListAustralia = True
-ListFrance = True
-ListGermany = True
-ListItaly = True
-ListKorea = True
-ListNetherlands = True
-ListRussia = True
-ListSpain = True
-ListTaiwan = True
-ListWorld = True
-ListUnknown = True
-ListSort = -3
-ListSortSecondary = 0
-ColorCompressed = True
-ColumnPlatform = True
-ColumnBanner = True
-ColumnNotes = True
-ColumnFileName = True
-ColumnID = True
-ColumnRegion = True
-ColumnSize = True
-ColumnState = False
-[Core]
-HLE_BS2 = False
-TimingVariance = 8
-CPUCore = 1
-Fastmem = True
-CPUThread = True
-DSPHLE = True
-SyncOnSkipIdle = True
-SyncGPU = False
-SyncGpuMaxDistance = 200000
-SyncGpuMinDistance = -200000
-SyncGpuOverclock = 1.00000000
-FPRF = False
-AccurateNaNs = False
-DefaultISO = 
-DVDRoot = 
-Apploader = 
-EnableCheats = True
-SelectedLanguage = 0
-OverrideGCLang = False
-DPL2Decoder = False
-TimeStretching = False
-RSHACK = False
-Latency = 0
-MemcardAPath = /home/$whoami/Desktop/slippi/Slippi-FM-r18_1/playback/User/GC/MemoryCardA.USA.raw
-MemcardBPath = /home/$whoami/Desktop/slippi/Slippi-FM-r18_1/playback/User/GC/MemoryCardB.USA.raw
-AgpCartAPath = 
-AgpCartBPath = 
-SlotA = 255
-SlotB = 10
-SerialPort1 = 255
-BBA_MAC = 
-SIDevice0 = 12
-AdapterRumble0 = False
-SimulateKonga0 = False
-SIDevice1 = 0
-AdapterRumble1 = False
-SimulateKonga1 = False
-SIDevice2 = 0
-AdapterRumble2 = False
-SimulateKonga2 = False
-SIDevice3 = 0
-AdapterRumble3 = False
-SimulateKonga3 = False
-WiiSDCard = False
-WiiKeyboard = False
-WiimoteContinuousScanning = False
-WiimoteEnableSpeaker = False
-RunCompareServer = False
-RunCompareClient = False
-EmulationSpeed = 1.00000000
-FrameSkip = 0x00000000
-Overclock = 1.00000000
-OverclockEnable = False
-GFXBackend = 
-GPUDeterminismMode = auto
-PerfMapDir = 
-EnableCustomRTC = False
-CustomRTCValue = 0x386d4380
-AllowAllNetplayVersions = False
-QoSEnabled = True
-AdapterWarning = True
-ShownLagReductionWarning = False
-[Movie]
-PauseMovie = False
-Author = 
-DumpFrames = True
-DumpFramesSilent = True
-ShowInputDisplay = False
-ShowRTC = False
-[DSP]
-EnableJIT = False
-DumpAudio = True
-DumpAudioSilent = True
-DumpUCode = False
-Backend = ALSA
-Volume = 24
-CaptureLog = False
-[Input]
-BackgroundInput = False
-[FifoPlayer]
-LoopReplay = True
-[Analytics]
-ID = 77340e4dd1ee0ecdb7e14ad946869752
-Enabled = False
-PermissionAsked = True
-[Network]
-SSLDumpRead = False
-SSLDumpWrite = False
-SSLVerifyCert = False
-SSLDumpRootCA = False
-SSLDumpPeerCert = False
-[BluetoothPassthrough]
-Enabled = False
-VID = -1
-PID = -1
-LinkKeys = 
-[Sysconf]
-SensorBarPosition = 1
-SensorBarSensitivity = 50331648
-SpeakerVolume = 88
-WiimoteMotor = True
-WiiLanguage = 1
-AspectRatio = 1
-Screensaver = 0
-[NetPlay]
-SelectedHostGame = SUPER SMASH BROS. Melee (GALE01, Revision 2)
-TraversalChoice = traversal
-Nickname = Player
-HostCode = 00000000
-ConnectPort = 2626
-HostPort = 2626
-ListenPort = 0
-NetWindowPosX = 390
-NetWindowPosY = 200
-NetWindowWidth = 768
-NetWindowHeight = 640
+[General]

```

Comment:
This file has is marked as completely different because of whitespace/newline changes, but it shouldn't really be a problem. I'll try to configure my git to use unix line endings next time.

Reply:
Actually I don't think I need anything from this file so I will just checkout master for this file.